src/osx/cocoa/combobox.mm

```diff
@@ -226,7 +226,23 @@ - (void)comboBoxSelectionDidChange:(NSNotification *)notification
 
 void wxNSComboBoxControl::InsertItem(int pos, const wxString& item)
 {
-    [m_comboBox insertItemWithObjectValue:wxCFStringRef( item , m_wxPeer->GetFont().GetEncoding() ).AsNSString() atIndex:pos];
+    NSComparator comp = ^(id obj1, id obj2)
+    {
+        return [obj1 caseInsensitiveCompare: obj2];
+    };
+    if ( m_wxPeer->HasFlag( wxCB_SORT ) )
+    {
+        wxCFStringRef itemLabel( item );
+        NSString * const cocoaStr = itemLabel.AsNSString();
+        NSArray *objectValues = m_comboBox.objectValues;
+        [m_comboBox insertItemWithObjectValue:cocoaStr
+                atIndex: [objectValues indexOfObject: cocoaStr
+                                       inSortedRange: NSMakeRange(0, objectValues.count)
+                                       options: NSBinarySearchingInsertionIndex
+                                       usingComparator: comp]];
+    }
+    else
+        [m_comboBox insertItemWithObjectValue:wxCFStringRef( item , m_wxPeer->GetFont().GetEncoding() ).AsNSString() atIndex:pos];
```

Comment:
Why not use `cocoaStr` for this one too? I just don't know, it's pretty minor, but I don't understand what reason could you possible have to duplicate this call rather than just do something like

```cpp
wxCFStringRef itemLabel(item);
NSString * const cocoaStr = itemLabel.AsNSString();
if ( m_wxPeer->HasFlag(wxCB_SORT) )
{
    ...
    pos = [objectValues indexOfObject: ...];
}
[m_comboBox insertItemWithObjectValue:cocoaStr atIndex:pos];
```
?

Reply:
@vadz,
I'm just follow notion of less intrusion.
The idea is to change only the code required to fix the problem and leave the rest alone.
This is the reason behind not changing this line.

It is an easy change and I can change it and push it, but just wanted to explain my reasoning behind leaving the original code "as-is".